content/docs/alerting/notifications.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Notification template reference
+sort_rank: 6
+---
+# Notification Template Reference
+
+Prometheus creates and sends alerts to the Alertmanager which then sends notifications out to different receivers based on their labels.
+A receiver can be one of many integrations including: Slack, PagerDuty, email, or a custom integration via the generic webhook interface.
+
+The notifications sent to receivers are constructed via templates. The Alertmanager comes with default templates but they can also be customized.
+To avoid confusion it's important to note that the Alertmanager templates differ from [templating in Prometheus](https://prometheus.io/docs/visualization/template_reference/), however Prometheus templating also includes the templating in alert rules labels/annotations.
+
+
+The Alertmanager's notification templates are based on the [Go templating](http://golang.org/pkg/text/template) system.
+Note that some fields are evaluated as text, and others as HTML which will affect escaping.
+
+# Data Structures
+
+## Data
+
+`Data` is the structure passed to notification templates and webhook pushes.
+
+| Name          | Type     | Notes    |
+| ------------- | ------------- | -------- |
+| Receiver | string | Defines the receiver's name that the notification will be sent to (slack, email etc.). |
+| Status | string | Defined as firing if at least one alert is firing, otherwise resolved. |
+| Alerts | [Alert](#alert) | List of alert objects ([see below](#alert)). |
+| GroupLabels | [KV](#kv) | The labels these alerts were grouped by. |
+| CommonLabels | [KV](#kv) | The labels common to all of the alerts. |
+| CommonAnnotations | [KV](#kv) | Set of common annotations to all of the alerts. Used for longer additional strings of information about the alert. |
+| ExternalURL | string | Backlink to the Alertmanager that sent the notification. |
+
+## Alert
+
+`Alert` holds one alert for notification templates.
+
+| Name          | Type     | Notes    |
+| ------------- | ------------- | -------- |
+| Status | string | Defines whether or not the alert is resolved or currently firing. |
+| Labels | [KV](#kv) | Specifies a set of labels to be attached to the alert. |
```

Comment:
There's still specifies here, this is a data structure you transverse not one you specify.